admin/app/components/solidus_admin/sidebar/item/component.rb

```diff
@@ -4,7 +4,10 @@
 class SolidusAdmin::Sidebar::Item::Component < SolidusAdmin::BaseComponent
   with_collection_parameter :item
 
-  def initialize(item:, url_helpers: solidus_admin_with_fallbacks)
+  def initialize(
+    item:,
+    url_helpers: Struct.new(:spree, :solidus_admin).new(spree, solidus_admin)
```

Comment:
Not needed right now, but might make sense in the future to also give access to `main_app` and other engines. 

A nice thing I just discovered is that `Rails.application.routes.mounted_helpers` is a module that will give access to all those helpers
 https://github.com/rails/rails/blob/495cbe9f8fec01ed3f2c0128dda39ca07bad00b3/actionpack/lib/action_dispatch/routing/route_set.rb#L473-L479

For sure I'll open a PR to include it in the base component

Reply:
Yeah, I thought about `main_app`, but as it's not already part of the available helpers and I'm not sure it's going to be needed, I preferred postponing it for the future.

> A nice thing I just discovered is that Rails.application.routes.mounted_helpers is a module that will give access to all those helpers

I spent some time looking for something like that but I didn't find it :laughing: It looks like it needs something else, though. `Rails.application.routes.mounted_helpers.solidus_admin` will raise an error on the console :thinking:

Reply:
Yes, because `Rails.application.routes.mounted_helpers` is a module that needs to be included in a class, that said I just tried on the BaseComponent and errors out if used before rendering (because it needs a reference to the controller). 

💯 for now we can stick to those two